library/src/scripts/routing/links/BackLink.tsx

```diff
@@ -115,7 +115,7 @@ export default function BackLink(props: IProps) {
         );
     }
 
-    return <div className={classNames(classes.root, props.className)}>{content}</div>;
+    return <div className={`backLink ${classNames(classes.root, props.className)}`}>{content}</div>;
```

Comment:
instead of making this into a template, just move "backLink" as a param to `claccNames`. I also takes strings.

Reply:
```suggestion
    return <div className={classNames('backLink', classes.root, props.className)}>{content}</div>;
```